pkg/cmd/cli/restore/logs.go

```diff
@@ -82,6 +83,12 @@ func (l *LogsOptions) Validate(f client.Factory) error {
 	}
 	l.client = c
 
-	_, err = l.client.ArkV1().Restores(f.Namespace()).Get(l.RestoreName, metav1.GetOptions{})
+	r, err := l.client.ArkV1().Restores(f.Namespace()).Get(l.RestoreName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	if r.Status.Phase != v1.RestorePhaseCompleted {
+		err = errors.Errorf("Restore %s is not yet complete.")
```

Comment:
I would say "unable to retrieve logs because restore is not yet complete." I don't think we need to include the name of the restore.